disk/smart-monitoring.sh

```diff
@@ -54,6 +62,37 @@ fi
 
 # Install needed tools
 install_if_not smartmontools
+
+# Test drives
+print_text_in_color "$ICyan" "Testing if all drives support smart monitoring and are healthy..."
+mapfile -t DRIVES <<< "$DRIVES"
+for drive in "${DRIVES[@]}"
+do
+    echo '#########################'
+    print_text_in_color "$ICyan" "Testing /dev/$drive"
+    if ! smartctl -a "/dev/$drive"
+    then
+        print_text_in_color "$IRed" "/dev/$drive doesn't support smart monitoring or isn't healthy"
```

Comment:
is this intended? first print then msg_box?

Reply:
yes, for the output log of the script this makes it much more readable. But I also think that showing the msg_box here makes sense -> I do both

Reply:
I could also reverse the order -> show msg_box first and then print_text_in_color. What do you think?

Reply:
Whatever. Doesn't matter to me.